src/main/java/io/cdap/plugin/successfactors/common/util/ResourceConstants.java

```diff
@@ -38,6 +39,7 @@ public enum ResourceConstants {
   ERR_CHECK_ADVANCED_PARAM(null, "err.check.advanced.parameter"),
   ERR_NOT_FOUND(null, "err.resource.not.found"),
   ERR_NO_COLUMN_FOUND(null, "err.no.column.found"),
+  ERR_UNSUPPORTED_ASSOCIATEDENTITY(null, "err.unsupported.associatedEntity"),
```

Comment:
ERR_UNSUPPORTED_ASSOCIATEDENTITY, this can be renamed to ERR_UNSUPPORTED_ASSOCIATED_ENTITY

Reply:
+1
also update the value to `err.unsupported.associated.entity`

Reply:
done